### DISCUSSION TITLE
After upgrading webdriverio to 8.32.2 I see that chrome driver is not fit to chrome browser version

### DISCUSSION THREAD

**Discussion by meirifrach23** [webdriverio/webdriverio#12329] [Q&A]
Hi,

I use webdriverio 8.32.2 to run my e2e web tests. I noticed that after the fix that support in the new chrome driver link that was changed by google (8.32.2), the driver that is being downloaded is no longer compatible with browser version. I have browser version 121 in my PC and the chrome driver version that was donloaded is 122, so all my tests were not event started.

This behavior is a bug?
I know that I can check what is the browser version installed on the PC and then set this version into "browserVersion" under "capabilities", but then we miss all the nice functionality of webdriverio 8 that automatically download proper version of chrome driver that fit with browser version.

 this is the log I get:

chromedriver: Starting ChromeDriver 122.0.6261.57 (87957e464700ccf3468df47ce741e0dc089e0544-refs/branch-heads/6261@{#853}) on port 54961
chromedriver: Remote connections are allowed by an allowlist (0.0.0.0).
chromedriver: Please see https://chromedriver.chromium.org/security-considerations for suggestions on keeping ChromeDriver safe.
chromedriver: ChromeDriver was started successfully.
@wdio/utils: Started Chromedriver vstable with params --port=54961 --allowed-origins=* --allowed-ips=0.0.0.0 in 2617ms on port 54961
ERROR@wdio/runner: Error: Failed to create session.
session not created: This version of ChromeDriver only supports Chrome version 122
Current browser version is 121.0.6167.185 with binary path C:\\\\Program Files (x86)\\\\Google\\\\Chrome\\\\Application\\\\chrome.exe
     at startWebDriverSession (file:///C:/Drop/Automation/node_modules/webdriver/build/utils.js:69:15)
     at processTicksAndRejections (node:internal/process/task_queues:95:5)
     at async Function.newSession (file:///C:/Drop/Automation/node_modules/webdriver/build/index.js:20:45)
     at async remote (file:///C:/Drop/Automation/node_modules/webdriverio/build/index.js:45:22)
     at async Runner._startSession (file:///C:***@wdio/runner/build/index.js:241:29)
     at async Runner._initSession (file:///C:***@wdio/runner/build/index.js:207:25)
     at async Runner.run (file:///C:***@wdio/runner/build/index.js:88:19)

Here is how the relevant configuration is looks like:

capabilities: [
  {
    browserName: "chrome",
    "goog:chromeOptions": {
      binary: "C:\\Program Files (x86)\Google\\Chrome\\Application\\chrome.exe"
    }
  }
]

**Comment by christian-bromann** [comment]
@meirifrach23 can you check and update to the latest WebdriverIO version?

I am not able to reproduce this. If I set the binary to an old Chrome version like this:

```ts
browserName: 'chrome',
'goog:chromeOptions': {
    binary: '/private/tmp/ff/chrome/mac_arm-119.0.6045.105/chrome-mac-arm64/Google Chrome for Testing.app/Contents/MacOS/Google Chrome for Testing'
}
```

WebdriverIO is still able to figure out to download the correct Chromedriver version. Any chance you can provide a reproducible example?

**Comment by meirifrach23** [comment]
I work with latest WebdriverIO version (8.32.2).
Maybe you cannot reproduce the issue because you work with "chrome for testing"?

I use regular chrome browser that installed in:
C:\Program Files (x86)\Google\Chrome\Application\chrome.exe

I noticed that the chrome driver file was not downloaded (usually I can find the chrome driver under: C:\Users\user\AppData\Local\Temp\chromedriver), and all I got is this error (as I show in my first message):

ERROR@wdio/runner: Error: Failed to create session.
session not created: This version of ChromeDriver only supports Chrome version 122
Current browser version is 121.0.6167.185 with binary path C:\\Program Files (x86)\\Google\\Chrome\\Application\\chrome.exe

![image](https://github.com/webdriverio/webdriverio/assets/29702690/2ddcc646-8e9c-4adb-9788-2015ba75fd7a)

**Reply by christian-bromann** [reply]
It doesn't matter if you use stock Chrome or Chrome for testing, my point was that I set a custom browser binary and WebdriverIO was able to resolve the version and download the right Chromedriver. Please provide a reproducible example if possible.

**Reply by selenium-stack** [reply]
@meirifrach23 i am also facing the same error. 
But i have one question why dont you have chromedriver in you package.json?
